tests/Unit/PointwiseFunctions/AnalyticSolutions/Burgers/Test_Step.cpp

```diff
@@ -0,0 +1,50 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <vector>  // IWYU pragma: keep
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Burgers/Step.hpp"
+#include "tests/Unit/PointwiseFunctions/AnalyticSolutions/Burgers/CheckBurgersSolution.hpp"
+#include "tests/Unit/TestCreation.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+// IWYU pragma: no_forward_declare Tensor
+
+SPECTRE_TEST_CASE("Unit.PointwiseFunctions.AnalyticSolutions.Burgers.Step",
+                  "[PointwiseFunctions][Unit]") {
+  test_creation<Burgers::Solutions::Step>(
```

Comment:
`test_creation` returns an object. You should test that it was set up correctly. All this is testing is that we can actually parse that string, but not that we parse it correctly.

Reply:
Done. Also added a new commit that adds similar checking of what was created by `test_creation` in several other tests that were missing this.